src/__mocks__/statusUpdate.js

```diff
@@ -0,0 +1,30 @@
+function clearElement(element) {
```

Comment:
- [ ] We usually use the mocks to mock API responses (or other external dependencies like databases, local storage), not for the parts of our application. 
You have created whole new functions in your mock files and these functions have nothing to do with your original functions.
Basically in your test files, you are testing the functions that you created in your mock files, not the functions that you used in your application. 
**Please write tests for the functions that you used in your application.**

Reply:
Hello Sinan, thank you for your review. :pray: 
I would like to clarify that the reason I mocked my functions is that they depend on external factors to work (such as user input). What I did was that I copied my existing functions into the mock files to be able to remove the external variables that I am not testing at the moment. Would you be so kind as to compare the mock functions to the mock versions?
You may realize they are actually the same functions just "stripped from their dependencies". I yet don't know how to test my functions with all of their dependencies. If it is paramount to stop using mocks, would you mind explaining to me how I can go about doing such tests? I would really appreciate it. Thank you very much for your time.

Reply:
Hi Alex, 
I noticed that, But one of the key requirements for this project is mocking these external dependencies. If you strip down these external dependencies, you may not notice possible errors that they can cause on your application. (Please check [the project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/tests_1.md#:~:text=Some%20of%20those,in%20the%20DOM. ) 

You can check these articles about mocking => [Mocking DOM](https://ms314006.github.io/unit-testing-for-dom-manipulation/),  [Mocking localStorage](https://nofluffweb.com/mock-local-storage-in-jest-tests), [Mocking](https://bholmes.dev/blog/mocking-browser-apis-fetch-localstorage-dates-the-easy-way-with-jest/), 

For this milestone, you only need to create tests for `add item` (createTask) and `delete item` (clearElement) functions. I am pretty sure you can quickly write tests for them :pray: